api/database/migrations/2025_01_02_144403_create_community_interest_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
```

Comment:
Could you add some uniqueness constraints to these tables?  To prevent duplicates of user->community and interest->stream.